docs/HowToPortTizenRTOnWiFiChipset.md

```diff
@@ -0,0 +1,141 @@
+# How to port TizenRT on new WiFi chipset
+
+There are two cases to consider towards porting TizenRT on a new WiFi chipset. The specific case depends on the vendor's decision to either use their proprietary supplicant codebase, or
+to directly use TizenRT's WiFi solution.
+
+Below are mentioned the two cases in detail. Additionally, instructions are provided to incorporate the WiFi chipset driver in TizenRT's code tree. Specifically, details are provided
+to interface driver to network stack, and to get/set WiFi configuration from a higher abstraction layer.
+ 
+## How to add external WiFi library to TizenRT
```

Comment:
Let's remove ```to TizenRT```. Here is only TizenRT so that we don't need to say it. Shorter is better for title I think.